x-pack/plugins/fleet/public/components/agent_enrollment_flyout/index.tsx

```diff
@@ -93,6 +95,32 @@ export const AgentEnrollmentFlyout: React.FunctionComponent<Props> = ({
     checkPolicyIsFleetServer();
   }, [policyId]);
 
+  const [isK8s, setIsK8s] = useState<'IS_LOADING' | 'IS_KUBERNETES' | 'IS_NOT_KUBERNETES'>(
```

Comment:
I think we can move this to it's own hook and use the same hook for standalone instructions